src/pages/billing/SpendReport/SpendReport.test.ts

```diff
@@ -182,8 +182,8 @@ describe('SpendReport', () => {
     })
     expect(screen.getByTestId('spend')).toHaveTextContent('$1,110.17*')
     expect(getSpendReport).toHaveBeenCalledTimes(2)
-    expect(getSpendReport).toHaveBeenNthCalledWith(1, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-03-02' })
-    expect(getSpendReport).toHaveBeenNthCalledWith(2, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-01-01' })
+    expect(getSpendReport).toHaveBeenNthCalledWith(1, { billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-03-02', aggregationKeys: ['Workspace~Category', 'Category'] })
```

Comment:
Probably worth adding a unit test case with an Azure billing project and ensuring that the workspace aggregation key is not included.